scripts/ci/compile_closure.sh

```diff
@@ -5,7 +5,7 @@
 # TODO(joeltine): Make strictMissingRequire an error when 
 # @suppress {missingRequire} works for it.
 
-java -Xmx1G -jar ../closure-compiler/build/compiler.jar \
+java -Xmx1G -jar ../closure-compiler/target/closure-compiler-1.0-SNAPSHOT.jar \
```

Comment:
This is because you're building from head. You can use the released compiler (from maven central) instead but when I set this up, I did this because
1. It was easier, and I was lazy
2. This lets us detect immediately if the compiler team lands a change that's going to break things in Closure, or if a PR in Closure is going to break when compiled with unreleased compiler changes.